dotnet/src/Connectors/Connectors.Ollama.UnitTests/Services/OllamaChatCompletionTests.cs

```diff
@@ -302,12 +304,6 @@ public async Task GetChatMessageContentsShouldAdvertiseToolAsync()
         Assert.NotNull(requestPayload.Tools);
         Assert.NotEmpty(requestPayload.Tools);
         Assert.Equal(1, requestPayload.Tools?.Count());
-        var firstTool = requestPayload.Tools?.First()!;
-        Assert.Equal("TestPlugin-TestFunction", firstTool.Function!.Name);
-        Assert.Single(firstTool.Function!.Parameters!.Properties!);
-        Assert.Equal("testInput", firstTool.Function!.Parameters!.Properties!.First().Key);
-        Assert.Equal("string", firstTool.Function!.Parameters!.Properties!.First().Value.Type);
-        Assert.Equal("testInput", firstTool.Function!.Parameters!.Required!.First());
 
         Assert.NotNull(message.ModelId);
```

Comment:
Any special reason why those assertions were removed, all should still pass.

Reply:
I'm not sure how it ever passed. Ollama's ChatRequest.Tools is a `List<object>`.